website/profile/utils.py

```diff
@@ -40,6 +40,7 @@ def serialize_user(user, node=None, admin=False, full=False):
     fullname = user.display_full_name(node=node)
     ret = {
         'id': str(user._primary_key),
+        'username': user.username,
```

Comment:
@leb2dg Lyndsy is right. This method takes an optional ```full``` argument. Let's use that and get the email for that serialized representation.